lib/engine/config/game/g_18_ms.rb

```diff
@@ -179,7 +179,17 @@ module G18MS
          "value":50,
          "revenue":15,
          "desc":"The owning Major Company may lay their cheapest available token for half price. This is not an extra token placement. This ability can only be used once, and using it does not close the private company.",
-         "sym":"M&M"
+         "sym":"M&M",
+         "abilities": [
+            {
+              "type": "token",
+              "owner_type":"corporation",
+              "hexes": [],
+              "price": 0.5,
```

Comment:
would it be better to call it a discount? what do the other things do?

Reply:
I have now added a new attribute 'discount' that is used for this purpose.